### PR TITLE
Fix parsing of class names like 'w-1/3' as used in selectors like '.w-1\/3'

### LINKED ISSUE #4609: Some TailwindCSS class names are being removed
## Bug Description

Some TailwindCSS class names such as `.w-1/3` are being removed.

## Expected Behaviour

Classes with `/` should be kept.

## Additional context

This is happening since the update on composer.lock here
https://github.com/ampproject/amp-wp/commit/ab513d451c2d1602a56576ac909ce59f7073dd24#diff-1da2c7edc898c70e5a79a9997c98cecc

### PR DESCRIPTION
**Pull request by westonruter**
## Summary

* Update PHP-CSS-Parser to [134f4e6...bc6ec74](https://github.com/sabberworm/PHP-CSS-Parser/compare/134f4e6...bc6ec74) on `master`. 
* Remove `patches/php-css-parser-138-extended.patch` (introduced in #4300) which was merged into `master`.
* Update patch to fix parsing CSS selectors which contain commas, originally in https://github.com/sabberworm/PHP-CSS-Parser/pull/138 but now updated at https://github.com/westonruter/PHP-CSS-Parser/pull/1.

Fixes #4609

## Checklist

- [x] My pull request is addressing an open issue (please create one otherwise).
- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

**Comment by westonruter** [comment]
Build for testing: [amp.zip](https://github.com/ampproject/amp-wp/files/4506330/amp.zip) (v1.5.4-alpha-20200420T213813Z-70cee0e15)

File: patches/php-css-parser-commit-10a2501.patch [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+---
+ .../CSS/RuleSet/DeclarationBlock.php          | 60 ++++++++++++++++++-
+ tests/Sabberworm/CSS/ParserTest.php           |  6 ++
+ tests/files/specificity.css                   |  4 +-
+ 3 files changed, 68 insertions(+), 2 deletions(-)
+
+diff --git a/lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php b/lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php
+index e18f5d8..26c2e12 100644
+--- a/lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php
++++ b/lib/Sabberworm/CSS/RuleSet/DeclarationBlock.php
```
Comment (schlessera) [inline patches/php-css-parser-commit-10a2501.patch:15]: Note that SabberWorm PHP CSS Parser seems to use PSR-2/PSR-12, not WPCS, so code style needs to be adapted if this is to be merged into the upstream package.

**Review by schlessera — APPROVED** [review]

**Review by pierlon — APPROVED** [review]
Otherwise from @schlessera's note above, this looks good :+1: .

**Comment by westonruter** [comment]
I'll keep this to the v1.6 release since updating the CSS parser could have big impacts.

**Pull request merged**